Monika After Story/game/script-compliments.rpy

```diff
@@ -888,6 +889,174 @@ label mas_compliment_humor_3:
+                    m 1esa "What shall we do today, [mas_get_player_nickname()]?"
+
+                    $ mas_gainAffection(0.75)#hug aff
+                    return
+                "Not right now.":
+                    #a lil sad
+                    $ mas_loseAffection()
+                    m 6ekc "...alright, [player]. Maybe later, then?"
+    #Base negative responses on monika_love label
+    elif mas_isMoniUpset():
+        $ mas_gainAffection()
+        python:
+            missed_quips_upset_cynical = [
+                "...did you really even think of me at all?",
+                "...were you really thinking about me even a little?",
+                "That didn't feel very sincere, [player].",
+                "I'm not sure if you really mean that, [player].",
+                "If you really mean that, then please spend some time with me today.",
+                "If you really meant that, then you'd spend more time with me when you're here.",
+                "You say that, but I don't think you mean it one bit, [player].",
+                "You say that, but I feel like you don't mean it at all, [player]."
+            ]
+
+            missed_quips_upset_hope = [
+                "Thank you for showing me you still care, [player].",
+                "It means a lot to me that you were thinking of me.",
+                "I'm really glad to hear that, [player].",
+                "That's really nice to hear, coming from you.",
+                "I'm happy you've been thinking of me, [player].",
+                "[player]... I'm glad I'm not the only one.",
+                "That means the world to me, [player].",
+                "That makes me feel a lot better, [player]."
+            ]
+
+        if _mas_getAffection() <= -50:
+            $ missed_quip_upset = renpy.random.choice(missed_quips_upset_cynical)
+            m 2rkc "..."
+            show monika 2ekd
+            $ renpy.say(m, missed_quip_upset)
+
+            if mas_getAbsenceLength() >= datetime.timedelta(days=3):
+                m 2ekd "...but at least you didn't forget about me."
+
+        else:
+            $ missed_quip_upset = renpy.random.choice(missed_quips_upset_hope)
+            m 2ekd "Thanks, [player]..."
+            show monika 2dka
+            $ renpy.say(m, missed_quip_upset)
+
+            if mas_getAbsenceLength() >= datetime.timedelta(days=3):
+                m 2ekd "Thank you for coming back. I was starting to worry that you'd forgotten me."
+                if renpy.random.randint(1,2) == 1:
+                    m 2eka "I...{w=0.5}I also really missed you."
+                else:
+                     m 2eka "I...{w=0.5}I really missed you too."
+            else:
+                if renpy.random.randint(1,2) == 1:
+                    m 2eka "I...{w=0.5}I also missed you."
+                else:
+                    m 2eka "I...{w=0.5}I missed you too."
+    elif mas_isMoniDis():
+        python:
+            missed_quips_dis = [
```

Comment:
Same with these.

Reply:
Change made in https://github.com/Monika-After-Story/MonikaModDev/pull/7419/commits/a70b95c814f202d80af1f694e15d841725713e31.